### PR TITLE
Fix #305 add boot.sh; link index.sh

### LINKED ISSUE #305: Simplify setting up a host in dev

### PR DESCRIPTION
**Pull request by robnagler**
- Support `curl http://v3.radia.run:2916/boot.sh | bash` for easy setup of VMs.

### CODE REVIEW

**Comment by robnagler** [comment]
Depends on #302 which sets TZ in tests.

**Comment by robnagler** [comment]
@Benbenbenin0 ping

File: tests/pkcli/build_data/1.out/srv/host/v3.radia.run/etc/systemd/system/docker_registry.service [hunk 1]
```diff
@@ -7,9 +7,9 @@ Requires=docker.service
 ExecStart=/srv/docker_registry/start
 ExecStartPre=/srv/docker_registry/remove
 ExecStop=-/srv/docker_registry/stop
+SyslogIdentifier=docker_registry
```
[the remark this answers is not in the document]
Comment (Benbenbenin0) [reply tests/pkcli/build_data/1.out/srv/host/v3.radia.run/etc/systemd/system/docker_registry.service:10]: Also present:
- tests/pkcli/build_data/1.out/srv/host/v4.radia.run/etc/systemd/system/sirepo.service
- tests/pkcli/build_data/1.out/srv/host/v4.radia.run/etc/systemd/system/sirepo_job_supervisor.service
- tests/pkcli/build_data/1.out/srv/host/v4.radia.run/etc/systemd/system/sirepo_jupyterhub.service
- tests/pkcli/build_data/1.out/srv/host/v5.radia.run/etc/systemd/system/mpi_worker.service
- tests/pkcli/build_data/1.out/srv/host/v9.radia.run/etc/systemd/system/jupyterhub.service
- tests/pkcli/build_data/1.out/srv/host/v9.radia.run/etc/systemd/system/rs_mariadb.service
- tests/pkcli/build_data/1.out/srv/host/v9.radia.run/etc/systemd/system/sirepo.service
- tests/pkcli/build_data/1.out/srv/host/v9.radia.run/etc/systemd/system/sirepo_job_supervisor.service
- tests/pkcli/build_data/1.out/srv/host/v9.radia.run/etc/systemd/system/sirepo_jupyterhub.service
- tests/pkcli/build_data/1.out/srv/host/v9.radia.run/etc/systemd/system/wordpress.service

**Review by Benbenbenin0 — APPROVED** [review]
Approving with fmt comment (likely unimportant). Don't fully understand code additions, assuming sufficient testing. Other observed changes:
- New templates and cmd function (rsconf_systemctl)
- Wrapped/Refactor code into methods/helper methods
- Refactor variables/fields into wider use and rename
- More use of pk tools
- New tests

File: tests/pkcli/build_data/1.out/srv/host/v3.radia.run/etc/systemd/system/docker_registry.service (hunk 1 above)
[the remark this answers is not in the document]
Comment (robnagler) [reply tests/pkcli/build_data/1.out/srv/host/v3.radia.run/etc/systemd/system/docker_registry.service]: Unfortunately, it was showing the wrong diffs... I merged #302 which was the vast majority of the changes. I needed those changes which were not yet in master (but should be now, but GitHub was confused, I think). 

This is the correct comparison: https://github.com/radiasoft/rsconf/compare/6596b44c..f8d1cce

> Marked change was 'SyslogIdentifier' moved above 'Restart'

The order is caused by jinja if statements. It was easier [to compose this way](https://github.com/radiasoft/rsconf/blob/6596b44ce1e35045669c548e2689ea92ea463810/rsconf/package_data/systemd/docker_unit.jinja#L10).

**Pull request merged**